### ISSUE TITLE
How to multi-select in "Generate Constructors" popup-menu

### ISSUE THREAD

**Issue by igbanam** [neoclide/coc-java#276]
What's the keystroke to select one of the boxes here?

`<cr>` just closes it and uses an empty selection to generate an empty constructor.

<img width="646" alt="Screenshot 2024-02-29 at 04 37 33" src="https://github.com/neoclide/coc-java/assets/390059/be44328c-f306-4e9b-a6ec-0e9bb5571b75">

**Comment by asmodeus812** [comment]
Try with tab, however i would like to suggest a config that auto selects all by default, since most of the time you start off of wanting all,, then maybe filter out some of those.

**Comment by asmodeus812** [comment]
Yeah, had you read my response, you would see that i suggest a config property that would allow to auto select all, which is obviously opt-in.
